ui.tests/test-module/specs/ruleeditor/customFunction.runtime.spec.js

```diff
@@ -86,4 +86,44 @@ describe('Form with custom functions configured in client lib', () => {
             });
         }
     })
+
+    it("should import formData on button click", () => {
+        if (cy.af.isLatestAddon() && toggle_array.includes("FT_FORMS-11541")) {
```

Comment:
Why is this FT required ?

Reply:
Passing globals object in scope param feature is behind this FT.